filebeat/_meta/common.full.p2.yml

```diff
@@ -231,6 +231,10 @@ filebeat.prospectors:
   # Redis AUTH password. Empty by default.
   #password: foobared
 
+#- type: udp
+
+  # Maximum size of each message sent over udp
+  #max_message_size: 1024
```

Comment:
Two things here:

* I think it would be more clear if it would say "Maximum size of the message received over UDP".
* 1024 seems low, UDP can go up to 65K. How about 10K, which should be large enough for most usecases but still small enough to not be noticeable in the mem usage?